x-pack/docs/en/security/authentication/remote-clusters-privileges-api-key.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[remote-clusters-privileges-api-key]]
+=== Configure roles and users
+
+// TODO: link "remote indices privileges"  to page that describes remote indices privileges
+
+To use a remote cluster for {ccr} or {ccs}, you need to create user roles with
+remote indices privileges on the local cluster.
```

Comment:
Should we add a link to the [doc](https://elasticsearch_98330.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/defining-roles.html#roles-remote-indices-priv) for remote indices privileges?